dotcom-rendering/src/web/experiments/tests/remove-business-liveblog-epics.ts

```diff
@@ -7,11 +7,11 @@ export const removeBusinessLiveblogEpics: ABTest = {
 	author: '@commercial-dev',
 	description:
 		'Test the commercial impact of removing contribution epics on business liveblogs',
-	audience: 0 / 100,
-	audienceOffset: 0 / 100,
-	audienceCriteria: 'Opt in',
+	audience: 20 / 100,
+	audienceOffset: 20 / 100,
+	audienceCriteria: 'Business liveblogs',
 	successMeasure: 'Ad revenue increases on business liveblogs',
-	canRun: () => true,
+	canRun: () => !!(window.guardian.config.page.contentType === 'LiveBlog'),
```

Comment:
Here you can omit the double negation, since the inner expression returns a boolean.
```suggestion
	canRun: () => window.guardian.config.page.contentType === 'LiveBlog',
```

Reply:
Good spot, thanks!